### DISCUSSION TITLE
Please publish example code for using mono space font size 2

### DISCUSSION THREAD

**Discussion by gert4git** [Bodmer/TFT_eSPI#2050] [Q&A - Fonts]
Hi,

New user of tft library here.

Could someone please publish example code how to use mono space font (size 2) ??

Thanks,
Gert

**Comment by Bodmer** [comment]
There are a large number of examples that use the fonts. Which one most closely fits your needs?

**Reply by gert4git** [reply]
Thanks! I am new to the ESP32 and the TFT library. I just need to displpay something in either courier or consolas (or something that looks close). Size 2 would be required.

Cheers,
Gert
